ropetest/refactor/suitestest.py

```diff
@@ -2,16 +2,16 @@
 
 import unittest
 
-from rope.base import ast
+from rope.base import astwrapper
 from rope.refactor import suites
 from ropetest import testutils
 
 
 class SuiteTest(unittest.TestCase):
-    def setUp(self):
+    def setUp(self):  # pylint: disable=useless-super-delegation
         super().setUp()
 
-    def tearDown(self):
+    def tearDown(self):  # pylint: disable=useless-super-delegation
         super().tearDown()
```

Comment:
These methods could probably just be removed; I don't think there's any legitimate reason to implement these only to call super.